### PR TITLE
Update doctrine/dbal

### PR DESCRIPTION
**Pull request by embaka**

### CODE REVIEW

**Comment by embaka** [comment]
update composer

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=7.4",
-        "doctrine/dbal": "^2.12",
+        "doctrine/dbal": "^3.4",
```
Comment (loevgaard) [inline composer.json:14]:
```suggestion
        "doctrine/dbal": "^2.12 || ^3.4",
```

**Comment by loevgaard** [comment]
Can you confirm this PR works in your application with `doctrine/dbal` `^3.4`?

**Comment by embaka** [comment]
not yet

**Comment by loevgaard** [comment]
Okay, let me know when you have tested it :)

**Comment by embaka** [comment]
I try to install with the forked bundles but nothing installs. I have another bundle that is installed from a fork. Am I missing something? My project is symphony 5.4 and last update of everything.

**Pull request closed without merging**